components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4832,6 +4833,13 @@ where
 
     fn on_ready_prepare_merge(&mut self, region: metapb::Region, state: MergeState) {
         fail_point!("on_apply_res_prepare_merge");
+        let source_range = CacheRange::from_region(&region);
+        info!(
+            "evict range due to prepare merge";
+            "source_range" => ?source_range,
+        );
+        self.ctx.engines.kv.evict_range(&source_range);
```

Comment:
How about evicting range in apply worker?
Evicting range in peer might be too late for `IngestSST` commands if we want to fix compatibility issue between IME and Lightning.

Reply:
Fine.